### ISSUE TITLE
Rich output structure example using gpt-3.5-turbo

### ISSUE THREAD

**Issue by cjfontec** [microsoft/guidance#314]
**Is your feature request related to a problem? Please describe.**
cant use gpt-3.5-turbo for the Rich output structure example

**Describe the solution you'd like**
an updated version of the Rich output structure example using gpt-3,5-turbo instead of davinci-003

**Describe alternatives you've considered**
modify the syntax of the example to use the user/assistant structure by myself (without success)

**Comment by recursionbane** [comment]
I don't think this is supported. Guidance still thinks gpt-3.5-turbo is a chat-optimized model, and does not allow structured tags within the ASSISTANT response section.
